src/test/models/Department.test.js

```diff
@@ -0,0 +1,19 @@
+import {
+  sequelize,
+  dataTypes,
+  checkModelName,
+  checkPropertyExists
+} from 'sequelize-test-helpers';
```

Comment:
why are you using sequelize test helpers here..

Reply:
To test the Database Models

Reply:
but i thought the TTL mentioned that we should not install extra packages??

Reply:
well, i guess its really needed. I looked into it

Reply:
@sa-ma i don't think you need to include id in the seed file, when you send a data to the db, uuid auto-generates it.

Reply:
I've resolved it. Thanks